package.json

```diff
@@ -90,6 +91,7 @@
     "@react-types/shared": "3.25.0",
     "@rtk-query/codegen-openapi": "^1.2.0",
     "@rtsao/plugin-proposal-class-properties": "7.0.1-patch.1",
+    "@stylistic/eslint-plugin-ts": "^2.9.0",
```

Comment:
do we actually use this anywhere? 🤔

Reply:
It's a peer dependency of the [grafana/eslint-config](https://github.com/grafana/eslint-config-grafana/blob/master/base.js#L63-L72).